src/tests/cmocka/test_nss_srv.c

```diff
@@ -1892,7 +1930,7 @@ void test_nss_getgrnam_space_sub(void **state)
 
     mock_input_user_or_group("space group");
     will_return(__wrap_sss_packet_get_cmd, SSS_NSS_GETGRNAM);
-    mock_fill_group_with_members(0);
+    mock_fill_group_with_members();
 
```

Comment:
Maybe it will be easier to read the test after inlining `mock_fill_group_with_members`
with `will_return_always(__wrap_sss_packet_get_body, WRAP_CALL_REAL);`